dbms/src/Server/Server.h

```diff
@@ -65,7 +65,7 @@ class Server : public BaseDaemon
     std::string getDefaultCorePath() const override;
 
 private:
-    std::unique_ptr<Context> global_context;
+    ContextPtr global_context;
```

Comment:
I think it is dangerous to change global_context from `std::unique_ptr` to be `shared_ptr`. It could easily cause
circular reference